transcom/mymove
# B-21976 Check feature flag for NTS/NTSR in office app

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21976)
## [INT PR](https://github.com/transcom/mymove/pull/14350)

## Summary

NTS/NTSR should not be on in prod, so we need to check the respective feature flags in the office app like we do elsewhere.

## TO TEST

1. Turn the feature flags for NTS and NTSR off
2. As a SC, attempt to add NTS and NTSR shipments to a move, and confirm you cannot
3. As a TOO, attempt to add NTS and NTSR shipments to a move, and confirm you cannot
